runtime/oti/j9consts.h

```diff
@@ -250,6 +250,9 @@ extern "C" {
 #define J9_RESOLVE_FLAG_JCL_CONSTANT_POOL 0x100
 #define J9_RESOLVE_FLAG_REDEFINE_CLASS 0x200
 #define J9_RESOLVE_FLAG_NO_CLASS_INIT 0x400
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define J9_RESOLVE_FLAG_FIELD_WITHER 0x800
```

Comment:
That's a very odd name.

Reply:
It a naming convention used in the earlier days of VT. (ie. getter, setter & wither).  
  
Ill change it to withfield for more clarity.